lib/OptionMenu/OptionMenu.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { node } from 'prop-types';
+import { Item } from './Item';
+
+function OptionMenu({ children }) {
+  return <div>{children}</div>;
```

Comment:
A bit off to just have a `div` here that doesn't do anything, but I think in the future we may want to style the container, and it means we can use the compound component pattern.

Reply:
Do we wanna have the option to give the div some classes?

Reply:
This component brings up a weird, point actually. It doesnt really do anything, just renders its children. 

Would we be better off just making `OptionMenu/MenuItem` a type of button or `DropdownAction`?

Reply:
It's definitely a smell just to have an empty div for the component, but the reasoning was to keep the GUI elements matched to the GCDS designs 👇

https://www.figma.com/file/66iSurBAx32DWrJymo9NKL/GCDS-%E2%80%93-Web-UI?node-id=321%3A0

In the designs the overarching element is `OptionMenu` and the sub element is `MenuItem`. Obviously these are open to change if they don't suit in the current form, but we should have a chat with @p34ks  before committing on our side.

I like the current pattern, as the designs suggest that further down the line we are going to have something like:
```
<OptionMenu>
  <OptionMenu.Heading>First Option Set</OptionMenu.Heading>
  <OptionMenu.MenuItem name="option 1" />
  <OptionMenu.MenuItem name="option 2" />
  <OptionMenu.MenuItem name="option 3" />
  <OptionMenu.Divider />
  <OptionMenu.MenuItem name="option 4" />
  <OptionMenu.Heading>Second Option Set</OptionMenu.Heading>
  <OptionMenu.MenuItem name="option 5" />
</OptionMenu>
```

I'm not sure about having the `MenuItem` as a generic button, as it has specific functionality that ties it to being a menu item (eg. the `selected`) prop.

My take away from the chat yesterday was that we were moving away from the `Dropdown` component rendering specific UI such as `DropdownAction`, and moving towards it just providing functionality to wrap a trigger and render arbitrary content when it is clicked. In this case, the arbitrary content is the `OptionMenu` and it's children.

Reply:
I really like how we would use it and feel fine with the fact that `OptionMenu` sort of does nothing. I think adding a `...rest` prop would be all we need, so we could still pass it classes or a test id.

What would this look like with the trigger?

Reply:
Sure, makes sense. Thanks for explaining!

💚 agree that it should be a `...rest` on the div though. I originally said just the className so my bad! 🙈